code/modules/urist/items/vgitems.dm

```diff
@@ -80,3 +80,73 @@
 	w_class = ITEM_SIZE_SMALL
 	obj_flags = OBJ_FLAG_CONDUCTIBLE
 	insultmsg = list("HONK!", "HELP, SHITCURITY!", "HELP MAINT", "SHITSEC", "I AM IN A CONSTANT STATE OF SUFFERING, EACH EON SPENT ON THIS PLANE OF EXISTENCE ECHOES ONLY PURE PAIN TO MY MORTAL COIL. GOD HAS DIED, AND HE HAS ONLY LEFT ME SUFFERING. ", "MY FUNNY BONE!")
+
+//lottery scratchers - west3436 of /vg/station13
+/obj/item/toy/lotto_ticket
+	name = "scratch-off lotto ticket"
+	desc = "A scratch-off lotto ticket."
+	icon = 'icons/urist/items/tgitems.dmi' //i see no reason to change this
+	w_class = ITEM_SIZE_TINY
+	var/revealed = FALSE
+	var/iswinner = FALSE
```

Comment:
This variable is redundant, as if it's a winning ticket, `winnings` is non-zero. I would remove this and instead simply check if `src.winnings` is not falsy while `revealed` is `TRUE`.

Reply:
ok

Reply:
Yes. But as I said, you could simply check if `winnings` is not falsy while `revealed` is `TRUE` (`if(T.revealed && T.winnings)`).
It'd also remove the additional `if` check here on LN124 in `vgitems`

Again, this isn't necessary, but seems odd to store a whole new variable for something that's checked once and all that data is already there

Reply:
ok fine